siliconcompiler/tools/verilator/import.py

```diff
@@ -0,0 +1,21 @@
+
+from .verilator import setup as setup_tool
+
+def setup(chip):
+    ''' Helper method to load configs specific to compile tasks.
+    '''
+
+    # Generic tool setup.
+    setup_tool(chip)
+
+    tool = 'verilator'
+    step = chip.get('arg','step')
+    index = chip.get('arg','index')
+    task = chip.get_task(step, index)
+    design = chip.top()
+
+    chip.add('tool', tool, 'task', task, 'option', step, index,  ['--lint-only', '--debug'])
```

Comment:
you use `set` in compile.py but `add` here. Any reason? or is this just something we need to fix in a later PR?

Reply:
Probably because I copy/pasted the task-specific logic out of the generic tool `setup` functions. I'll make them both use `add`.